index.js

```diff
@@ -1,250 +0,0 @@
-      console.log(answerText, midNumber);
-
-      // console.log(cheatDetector);
-      tryAgain = await ask(answerReply);
-
-      
-      if (tryAgain === "Y" || tryAgain === "y") {
-        // console.log(cheatDetector);
-
-        celebrate();
-      } else if (tryAgain === "N" || tryAgain === "n") {
-        if (cheatDetector.includes(midNumber)) {
-          console.log("but that contradicts what you said earlier!");
-          process.exit();
-        }
-        cheatDetector.push(midNumber);
-
-        secondSignal = await ask(secondGuess);
-        //put it while loop then a looping function3
-        if (secondSignal === "H" || secondSignal === "h") {
-          higher();
-        }
-        if (secondSignal === "L" || secondSignal === "l") {
-          lower();
-        }
-      }
-    }
-  
-}
-
-
-// player guess
-//1) player choose range
-//2) comp choose number whithin the range
-// 3) comp ask the player to guess
-// 4) in coorect end if wrong tell h or l
-
-
-//////////////////////
-///// Game 2 Start////
-//////////////////////
-async function playerGuess() {
-  lowNumber = await lowLimit();
-  lowNumber = +lowNumber;
-  highNumber = await highLimit();
-  highNumber = +highNumber;
-  factor = lowNumber;
-
- // console.log("low num" + lowNumber);
- // console.log("high num" + highNumber);
-
-  while (factor <= highNumber) {
-    numberRange.push(factor);
-    factor++;
-  }
-  //console.log(numberRange);
-  let guessNumber = Math.random(numberRange);
-  //console.log(guessNumber);
-  guessNumber = Math.round(guessNumber * 10);
-  //console.log(guessNumber);
-  guessNumber = numberRange[guessNumber];
-  //console.log(guessNumber);
-
-  playerAnswer = await ask(
-    `I chose a number between ${lowNumber} and ${highNumber}, Try to guess it.\n`
-  );
-  if (isNaN(playerAnswer)) {
-    playerAnswer = await ask("Guess again, But guess a NUMBER this time!\n");
-  }
-  while (playerAnswer != guessNumber) {
-    console.log("Nope your guess was wrong!");
-
-    if (playerAnswer < guessNumber) {
-      counter++;
-      playerAnswer = await ask("Guess again, But guess higher this time!\n");
-      if (isNaN(playerAnswer)) {
-        playerAnswer = await ask(
-          "Guess again, But guess a NUMBER this time!\n"
-        );
-      }
-    }
-
-    if (playerAnswer > guessNumber) {
-      counter++;
-      playerAnswer = await ask("Guess again, But guess lower this time!\n");
-      if (isNaN(playerAnswer)) {
-        playerAnswer = await ask(
-          "Guess again, But guess a NUMBER this time!\n"
-        );
-      }
-    }
-  }
-  if (playerAnswer == guessNumber) {
-    console.log(`Yay you got it the answer was ${guessNumber}\nyou got it in ${counter} steps.!!!!\n`);
-    
-  }
-  process.exit();
-}
-async function choose(){
-  player = await ask(`Choose a game mode:\nIf you want to guess press: P\n\nIf you want the computer to guess press: C\n`)
-  console.log(player);
-if (player == 'p' || player == 'P'){
-  playerGuess();
```

Comment:
There's a few instances where you may be able to save yourself some typing by doing some sanitizing to the human inputs. For instance, instead of looking for ` 'p' || 'P' ` on line 243 - you could add `.toUpperCase()` on the variables, in this case the player variable.